src/validation/custom.validation.ts

```diff
@@ -126,6 +126,17 @@ export const checkIdentityDateFields = (dayStr: string = "", monthStr: string =
   }
 };
 
+export const checkStartDateBeforeDOB = (startDay: string, startMonth: string, startYear: string, dobDay: string, dobMonth: string, dobYear: string) => {
```

Comment:
Jitin has a general function to compare a first and second date. We'll need to define checkStartDateBeforeDate in terms of that. Depending on merge order we'll need to fix that up.